packages/core/android/build.gradle

```diff
@@ -33,6 +33,7 @@ if (isNewArchitectureEnabled()) {
 }
 
 android {
+    namespace = "com.airbnb.android.react.lottie"
```

Comment:
@matinzd What do you think of using something like instead? https://github.com/software-mansion/react-native-gesture-handler/blob/9be7bc38d529473f27ba03ca61444b0484c1f28f/android/build.gradle#L173 Just to avoid making the library incompatible for everyone else not using Gradle 7 yet.

Reply:
Yeah I was thinking the same? How about the manifest itself?

Reply:
I think that prop should be ignored in AGP 8 and we can just leave that value. That's hos other packages are solving that at the moment.

Reply:
Yeah makes sense. I also removed `INTERNET` permission from it.